unix/w0vncserver/portals/RemoteDesktop.cxx

```diff
@@ -20,6 +20,9 @@
 
 static core::LogWriter vlog("RemoteDesktop");
 
+core::BoolParameter localcursor("localcursor", "[EXPERIMENTAL] Render cursor locally",
+                                 false);
```

Comment:
Probably best to include something in the setting name as well to make sure people realise this is a temporary setting.